website/static/js/nodesDelete.js

```diff
@@ -245,28 +252,23 @@ NodesDeleteViewModel.prototype.confirmChanges =  function() {
         return node.changed;
     });
 
-    if ($('#bbConfirmTextDelete').val() === this.confirmationString) {
-        if (nodesChanged.length <= 100) {
-            $osf.block('Deleting Project');
-            batchNodesDelete(nodesChanged.reverse()).then(function () {
-                self.page(self.WARNING);
-            }).fail(function (xhr) {
-                $osf.unblock();
-                var errorMessage = 'Unable to delete project';
-                if (xhr.responseJSON && xhr.responseJSON.errors) {
-                    errorMessage = xhr.responseJSON.errors[0].detail;
-                }
-                $osf.growl('Problem deleting project', errorMessage);
-                Raven.captureMessage('Could not batch delete projects.');
-                self.clear();
-                $('#nodesDelete').modal('hide');
-            }).always(function() {
-                $osf.unblock();
-            });
-        }
-    }
-    else {
-        $osf.growl('Verification failed', 'Strings did not match');
+    if (nodesChanged.length <= 100) {
+        $osf.block('Deleting Project');
+        batchNodesDelete(nodesChanged.reverse(), self.nodeType).then(function () {
+            self.page(self.WARNING);
+        }).fail(function (xhr) {
+            $osf.unblock();
+            var errorMessage = 'Unable to delete project';
+            if (xhr.responseJSON && xhr.responseJSON.errors) {
+                errorMessage = xhr.responseJSON.errors[0].detail;
+            }
+            $osf.growl('Problem deleting project', errorMessage);
```

Comment:
Growl message says problem deleting project, regardless of whether this is a project or a component.